addons/im_livechat/static/tests/tours/im_livechat_history_back_and_forth.js

```diff
@@ -35,7 +35,7 @@ registry.category("web_tour.tours").add("im_livechat_history_back_and_forth_tour
             run: "click",
         },
         {
-            trigger: ".o-mail-DiscussSidebar-item:contains(Visitor).o-active",
+            trigger: ".text-truncate:contains(Visitor)",
```

Comment:
.text-truncate is not a very descriptive/reliable selector.
Can't we have a target a little bit more functionally significant?